debian/changelog

```diff
@@ -1,3 +1,9 @@
+opx-nas-interface (5.20.3+opx15) unstable; urgency=medium
+
+  * Bugfix: fix opx-config-fanout code
```

Comment:
Please be clearer about what you're fixing.

Reply:
opx-config-fanout can current handle only interface name as argument, if ifindex is given it gives the following error:
root@OPX:~# opx-config-fanout --port 2 --mode 4x1 --speed 10g
Traceback (most recent call last):
  File "/usr/bin/opx-config-fanout", line 83, in <module>
    fp_obj = nas_fp_cache.get(front_port[0])
NameError: name 'front_port' is not defined

Will update the changelog message

Reply:
Please also make sure that you have a Github issue attached to this pull request, so that we know what bug you're fixing. **If a Github issue does not exist, please make one.**

Reply:
ok sure.